[2022-02-09 08:59:51,570] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-12-01T23:00:00+00:00 [queued]>
[2022-02-09 08:59:51,581] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-12-01T23:00:00+00:00 [queued]>
[2022-02-09 08:59:51,582] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:59:51,582] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 08:59:51,582] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:59:51,601] {taskinstance.py:1259} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2020-12-01 23:00:00+00:00
[2022-02-09 08:59:51,609] {standard_task_runner.py:52} INFO - Started process 308 to run task
[2022-02-09 08:59:51,617] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'bigquery_external_table_task', 'scheduled__2020-12-01T23:00:00+00:00', '--job-id', '480', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingestee_gcs_dag.py', '--cfg-path', '/tmp/tmp9xrh06vv', '--error-file', '/tmp/tmpsj22mmj3']
[2022-02-09 08:59:51,618] {standard_task_runner.py:77} INFO - Job 480: Subtask bigquery_external_table_task
[2022-02-09 08:59:51,698] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-12-01T23:00:00+00:00 [running]> on host cc021fb126ed
[2022-02-09 08:59:51,747] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 08:59:51,772] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=bigquery_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2020-12-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-12-01T23:00:00+00:00
[2022-02-09 08:59:51,775] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-02-09 08:59:52,836] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=bigquery_external_table_task, execution_date=20201201T230000, start_date=20220209T085951, end_date=20220209T085952
[2022-02-09 08:59:52,875] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 08:59:52,917] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-09 09:15:32,200] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-12-01T23:00:00+00:00 [queued]>
[2022-02-09 09:15:32,213] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-12-01T23:00:00+00:00 [queued]>
[2022-02-09 09:15:32,213] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:15:32,213] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:15:32,213] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:15:32,226] {taskinstance.py:1259} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2020-12-01 23:00:00+00:00
[2022-02-09 09:15:32,232] {standard_task_runner.py:52} INFO - Started process 440 to run task
[2022-02-09 09:15:32,239] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'bigquery_external_table_task', 'scheduled__2020-12-01T23:00:00+00:00', '--job-id', '580', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingesteee_gcs_dag.py', '--cfg-path', '/tmp/tmp3ahk6xnf', '--error-file', '/tmp/tmp6luqvx6x']
[2022-02-09 09:15:32,240] {standard_task_runner.py:77} INFO - Job 580: Subtask bigquery_external_table_task
[2022-02-09 09:15:32,313] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2020-12-01T23:00:00+00:00 [running]> on host 75953ff4f3d5
[2022-02-09 09:15:32,372] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:15:32,403] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=bigquery_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2020-12-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-12-01T23:00:00+00:00
[2022-02-09 09:15:32,405] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-02-09 09:15:33,050] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=bigquery_external_table_task, execution_date=20201201T230000, start_date=20220209T091532, end_date=20220209T091533
[2022-02-09 09:15:33,095] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:15:33,131] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
